src/components/Table/TableHead/FilterHeaderRow/_filter-header-row.scss

```diff
@@ -47,6 +47,15 @@
     padding-right: $spacing-03;
   }
 
+  .#{$iot-prefix}--filterheader-multiselect {
```

Comment:
Could you add comments why this is needed?

Reply:
Added